oximeter/impl/src/schema/codegen.rs

```diff
@@ -74,6 +74,41 @@ fn emit_target(schema: &TimeseriesSchema) -> TokenStream {
     emit_one(FieldSource::Target, schema)
 }
 
+/// Return true if all the fields in the schema are `Copy`.
+fn fields_are_copyable<'a>(
```

Comment:
Can we exhaustively match again the `FieldType` enum here? I'm worried about someone adding a new non-copy variant to `FieldType` that isn't copyable in the future.

Reply:
Yeah, good idea. Will do.

Reply:
Done in 35432b747